src/content/blog/2024_qcoder.md

```diff
@@ -0,0 +1,163 @@
+Shor's algorithm, for example, can factor integers with exponentially fewer queries than known classical algorithms.
+While learning about these algorithms is exciting, it can be challenging to stay motivated—especially for those without an academic background who are learning just for fun.
+Textbooks and academic papers are often too complex for beginners, making it hard to follow along.
+Even if you succeed in understanding and implementing an algorithm, running it on today’s quantum computers in scalable settings is typically impossible.
+
+In response to these challenges, we have developed <a href="https://www.qcoder.jp/" target="_blank">**QCoder**</a>, a platform for quantum competitive programming.
+QCoder is the quantum counterpart to conventional competitive programming platforms such as Codeforces, Google Code Jam, and ICPC.
+It allows users to learn quantum algorithms and practice quantum programming simply by participating in competitions.
+
+# Getting Started
+
+## Step 1: Environment Setup
+we currently support <a href="https://github.com/Qiskit/qiskit/" target="_blank">Qiskit</a> as the available quantum programming language.
+For the best coding experience, we recommend setting up a local Qiskit environment on your computer. This setup will provide useful features such as autocompletion and linting to improve productivity and code quality.
+
+To get started, please refer to the <a href="https://docs.quantum.ibm.com/guides/" target="_blank">official Qiskit documentation</a> for detailed installation instructions and guidelines.
+
+## Step 2: User Registratoin
+To participate in contests and submit your solutions, you need to sign up via the <a href="https://www.qcoder.jp/auth/signup" target="_blank">registration page</a>.
+
+## Step 3: Dive In and Enjoy!
+You now have access to all the content on QCoder. We suggest starting by participating in past contests. While you may find some parts difficult initially, you'll gain a deeper understanding as you solve problems and refer to the provided editorials. QCoder offers contests for everyone, from complete beginners to seasoned experts.
+
+# Sample Problem
+To help you get started, let's walk through a sample problem. Please note that this is just one example, and we offer both easier and more challenging problems. You can explore and choose the one that suits your level by visiting <a href="https://www.qcoder.jp/" target="_blank">QCoder</a>.
+
+## Problem Statement
+You are given an integer $n$, representing the number of qubits.
+Your task is to implement a quantum circuit with minimal circuit depth that prepares a <a href="https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state" target="_blank">GHZ state</a> using $n$ qubits.
+
+The GHZ state is defined as
+$$
+\begin{equation}
+\ket{\mathrm{GHZ}} = \frac{1}{\sqrt{2}} (\ket{0...0}_n - \ket{1...1}_n) \nonumber
+\end{equation}.
+$$
```

Comment:
Note that when using `$$` to delimit an equation, the `\begin{equation}`/`\end{equation}` delimiters are not needed. This applies for the rest of the file as well.
```suggestion
$$
\ket{\mathrm{GHZ}} = \frac{1}{\sqrt{2}} (\ket{0\cdots0}_n - \ket{1\cdots1}_n).
$$
```

Reply:
Thank you for the tip! I have commited the suggestion and applied to all other equations: https://github.com/unitaryfund/unitary.fund/pull/553/commits/d184bf1d69e0cd62143b5ca0e6966f0895cc8dec